gokart/file_processor.py

```diff
@@ -280,6 +280,7 @@ def make_file_processor(file_path: str, store_index_in_feather: bool) -> FilePro
         '.feather': FeatherFileProcessor(store_index_in_feather=store_index_in_feather),
         '.png': BinaryFileProcessor(),
         '.jpg': BinaryFileProcessor(),
+        '.ini': TextFileProcessor(),
```

Comment:
[nits]

I'd like to put this line below `'.txt': TextFileProcessor(),` , as continue to TextFileProcessor